src/main/java/pl/orki/hackathon/webapp/venue/control/VenueService.java

```diff
@@ -11,20 +11,16 @@
 @Service
 public class VenueService {
 
-    private final UserRepository userRepository;
     private final VenueRepository venueRepository;
+    private final UserService userService;
 
-    public VenueService(UserRepository userRepository, VenueRepository venueRepository) {
-        this.userRepository = userRepository;
+    public VenueService(VenueRepository venueRepository, UserService userService) {
         this.venueRepository = venueRepository;
+        this.userService = userService;
     }
 
     @Transactional
-    public Optional<Venue> createVenue(Venue venue, Long userId) {
-        return userRepository.findById(userId)
-                .map(user -> {
-                    venue.setUser(user);
-                    return venueRepository.save(venue);
-                });
+    public Optional<Venue> createVenue(Venue venue) {
+        return Optional.of(venueRepository.save(venue));
```

Comment:
I think it's not needed to wrapping `Venue` with `Optional` - there's no way that venue won't be created, because of not finding some data.